signup-ui/src/components/ui/landing-footer.tsx

```
@@ -0,0 +1,48 @@
+          alt="Veridonia Logo" 
+          className="h-6 w-6"
+        />
+        <span className="text-sm text-primary">
+          {t("copyright")}
+        </span>
+      </div>
+
+      {/* Right side - Links */}
+      <div className="flex flex-wrap items-center justify-center gap-4 text-sm">
+        <a
+          target="_blank"
```

Biome (2.3.13)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)